grafana/risingwave-dev-dashboard.dashboard.py

```diff
@@ -3372,32 +3372,62 @@ def section_iceberg_metrics(outer_panels):
+                    "Write Qps Of Iceberg Writer",
+                    "iceberg write qps",
                     [
                         panels.target(
-                            f"{metric('iceberg_file_appender_write_qps')}",
+                            f"{metric('iceberg_write_qps')}",
                            "{{executor_id}} @ {{sink_id}}",
                         ),
                     ]
                 ),
                 panels.timeseries_latency(
-                    "Write latency Of Iceberg File Appender",
+                    "Write Latency Of Iceberg Writer",
                     "",
                     [
                         *quantile(
                             lambda quantile, legend: panels.target(
-                                f"histogram_quantile({quantile}, sum(rate({metric('iceberg_file_appender_write_latency_bucket')}[$__rate_interval])) by (le, sink_id))",
+                                f"histogram_quantile({quantile}, sum(rate({metric('iceberg_write_latency_bucket')}[$__rate_interval])) by (le, sink_id))",
                                 f"p{legend}" + " @ {{sink_id}}",
                             ),
                             [50, 99, "max"],
                         ),
                         panels.target(
-                            f"sum by(le, sink_id)(rate({metric('iceberg_file_appender_write_latency_sum')}[$__rate_interval])) / sum by(le, type, job, instance) (rate({metric('iceberg_file_appender_write_latency_count')}[$__rate_interval]))",
+                            f"sum by(le, sink_id)(rate({metric('iceberg_write_latency_sum')}[$__rate_interval])) / sum by(le, type, job, instance) (rate({metric('iceberg_write_latency_count')}[$__rate_interval]))",
                             "avg @ {{sink_id}}",
                         ),
                     ],
                 ),
+                panels.timeseries_count(
+                    "Iceberg rolling unfushed data file",
+                    "",
+                    [
+                        panels.target(
+                            f"{metric('iceberg_rolling_unfushed_data_file')}",
+                           "{{executor_id}} @ {{sink_id}}",
+                        ),
+                    ]
+                ),
+                panels.timeseries_count(
+                    "Iceberg position delete cache num",
+                    "",
+                    [
+                        panels.target(
+                            f"{metric('iceberg_position_delete_cache_num')}",
+                           "{{executor_id}} @ {{sink_id}}",
+                        ),
+                    ]
+                ),
+                panels.timeseries_count(
+                    "Iceberg partition num",
+                    "",
+                    [
+                        panels.target(
+                            f"{metric('iceberg_partition_num')}",
+                           "{{executor_id}} @ {{sink_id}}",
```

Comment:
> what is the exec id in RW? I suggest using actor id instead

I think it means https://github.com/risingwavelabs/risingwave/blob/c8d351b81458507fe31e3ce86b8fce87284370ca/src/stream/src/executor/mod.rs#L172, because the sink metrics is per executor, executor id is used to identify it.🤔

Why not use, cc @wenym1 
https://github.com/risingwavelabs/risingwave/blob/f93175c1973e0e5aeb54399e833bcad7d06c4e7f/src/stream/src/task/stream_manager.rs#L131

Reply:
Just to be compatible with the code before I introduce this refactor, because before I introduce this refactor, the sink metric was using the executor id to form a identity string as the label. 

I think we can change to directly using the actor id or executor id, but we should remember to change the grafana dashboard at the meantime.

Reply:
Let's refactor them all in a separate PR.